### DISCUSSION TITLE
When installing on windows 11 arm I get amd64 version

### DISCUSSION THREAD

**Discussion by rwidmark** [JanDeDobbeleer/oh-my-posh#3354] [Troubleshoot]
Hi,
I did install oh my posh on windows 11 arm and I did get the amd64 version when I did run:
Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://ohmyposh.dev/install.ps1'))

How can I revert it / change it to windows 11 arm version?

**Comment by JanDeDobbeleer** [comment]
Download and install the correct installer from the releases page will replace the incorrect one. But I'll need you to see why it fetches the wrong one.

**Reply by JanDeDobbeleer** [reply]
What's the output of `Get-CimInstance -Class Win32_Processor -Property Architecture).Architecture`?

**Reply by rwidmark** [reply]
I did just download it trough powershell at windows 11 arm with powershell 7.3.1 arm version downloaded from Microsoft Store.
PS C:\Users\rstolpe> (Get-CimInstance -Class Win32_Processor -Property Architecture).Architecture
12
12
12
12
